tests/utils/mocked_pagers.rs

```diff
@@ -17,10 +17,20 @@ fn get_mocked_pagers_dir() -> PathBuf {
 /// On Unix: 'most' -> 'most'
 /// On Windows: 'most' -> 'most.bat'
 pub fn from(base: &str) -> String {
-    if cfg!(windows) {
-        format!("{}.bat", base)
-    } else {
-        String::from(base)
+    let mut cmd_and_args = base.split(" ");
+    let suffix = if cfg!(windows) { ".bat" } else { "" };
```

Comment:
Since we already depend on `shell_words::split` I think this code would end up simpler if we used it here too? If it ends up being worse, we can keep this as it is.

Reply:
Done, refactored it a bit further as well.